src/components/AddItem.js

```diff
@@ -1,7 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { writeToFirestore } from '../lib/firebase';
 
 const AddItem = () => {
-  return <h1>THIS IS THE ADD ITEM</h1>;
+  let [name, setName] = useState('');
+  let [frequency, setFrequency] = useState(7);
+
+  function handleSubmitForm(event) {
+    event.preventDefault();
+    if (name.length > 0) {
+      writeToFirestore('put-token-here', {
+        name,
+        frequency,
+        lastPurchaseDate: null,
+      });
+      alert(`${name} has been successfully added to your list.`);
+      setName('');
+    }
+  }
+
+  return (
+    <form onSubmit={event => handleSubmitForm(event)}>
+      <label htmlFor="add-shopping-list-item">Item Name</label>
+      <input
+        required
+        value={name}
+        onChange={e => setName(e.target.value)}
+        id="add-shopping-list-item"
+      />
+      <fieldset>
+        <legend>How soon will you buy this again?</legend>
+        <div>
+          {frequency === 7 ? (
+            <input
+              onChange={e => setFrequency(parseInt(e.target.value))}
+              type="radio"
+              id="soon"
+              name="frequency"
+              value={7}
+              checked
+            />
+          ) : (
```

Comment:
Is the use of the ternary to assist with the default selecting of the _"Soon"_ option? If so, I think this can be removed, as the `defaultChecked` property appears to be able to handle this feature on its own.